src-java/projectfloodlight/openflowj/build.gradle

```diff
@@ -46,9 +46,9 @@ task generateSources(type: Exec, dependsOn: cloneLoxigenRepo) {
         commandLine 'make', 'java'
     }
     else {
-        def uid = new UnixSystem().getUid()
-        def gid = new UnixSystem().getGid()
-        commandLine 'docker', 'run', '--rm', '-v', "$workingDir:$workingDir", '-w', "$workingDir", '-u', "$uid:$gid", 'kilda/base-ubuntu', 'make', 'java'
+//        def uid = new UnixSystem().getUid()
```

Comment:
what's wrong with that?

Reply:
ops, accidentally committed from my windows machine -_-
reverted changes in this file